### PR TITLE
tellico: init at 3.3.0

### PR DESCRIPTION
**Pull request by numkem**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/misc/tellico/default.nix [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    libksane
+    cmake
+    exempi
+    extra-cmake-modules
+    libcdio
+    kdeApplications.libkcddb
+    poppler
+  ];
+
+  meta = {
```
Comment (symphorien) [inline pkgs/applications/misc/tellico/default.nix:61]: can you add a homepage ?

Comment (numkem) [reply pkgs/applications/misc/tellico/default.nix:61]: Done!

**Comment by symphorien** [comment]
Looks good to me. The application builds and runs fine (I mean a window shows up).
I get this warning
```
kf5.kdbusaddons: Can not find 'kdeinit5' executable at  "/nix/store/m8grb7wlhjm5prhil3fbw5imr6byiihk-bash-interactive-4.4-p23/bin:/nix/store/sicn6b10ixbs6435d7dr630i2hxphfpr-patchelf-0.9/bin:/nix/store/k930l9lckxjf4zmafv4aapvdbd2sibmj-gcc-wrapper-9.3.0/bin:/nix/store/52x4908vr922dhnxz1i5rfnrsq244vzc-gcc-9.3.0/bin:/nix/store/g5lavnivfasac7m15g3mh9lhmhsr5vl4-glibc-2.30-bin/bin:/nix/store/ca9mkrf8sa8md8pv61jslhcnfk9mmg4p-coreutils-8.31/bin:/nix/store/3gd7zz0s80k50amwzj3015ddz0sd9inl-binutils-wrapper-2.31.1/bin:/nix/store/3b3ighb83nhifa1v4n7855hlbdl1mhf9-binutils-2.31.1/bin:/nix/store/g5lavnivfasac7m15g3mh9lhmhsr5vl4-glibc-2.30-bin/bin:/nix/store/ca9mkrf8sa8md8pv61jslhcnfk9mmg4p-coreutils-8.31/bin:/nix/store/nkmf179c1v5vis7v95dv05vi7r3sk28g-tellico/bin:/nix/store/ca9mkrf8sa8md8pv61jslhcnfk9mmg4p-coreutils-8.31/bin:/nix/store/hg3albf7g05ljfqrfjhd58rblimrp6ph-findutils-4.7.0/bin:/nix/store/k7rakfl1ld9ycfm1zzszly0161h6jlcp-diffutils-3.7/bin:/nix/store/jpqlmf3wqg281j8fdz50kjl525pfsxjc-gnused-4.8/bin:/nix/store/8pajzfyqx1v7dz1znrnrc4pqj5rmnx24-gnugrep-3.4/bin:/nix/store/miwvn81sgbbcq5bfglr6v3pwchgsd00c-gawk-5.0.1/bin:/nix/store/0s93dpvxn7vaaah3n1lgz0q0xqdfsbkj-gnutar-1.32/bin:/nix/store/6yqb5dfmlsx0zr0q8xzi8v4wbrzmvwpf-gzip-1.10/bin:/nix/store/hgp00a1rgwqxjcrkqhbjrfb5df7wjhl5-bzip2-1.0.6.0.1-bin/bin:/nix/store/998ykzc5mazwak08hxayqmyqkyz18fcb-gnumake-4.2.1/bin:/nix/store/kgp3vq8l9yb8mzghbw83kyr3f26yqvsz-bash-4.4-p23/bin:/nix/store/c8nzpy2f2sz0w3mp4x8gkpn4hd4l2420-patch-2.7.6/bin:/nix/store/6ca524vb3yg59ag3khm0010591rny9v1-xz-5.2.5-bin/bin:/nix/store/6ahbx5rlsk68kzbyzhzk74m4ki99wvr1-nix-2.3.4/bin:/nix/store/jw3apqr03z5812anw5c8rdp5iag4n0p2-git-2.25.3/bin:/nix/store/nlgrx0cl1l87nx0w660lb3jksn9jdca9-python3-3.7.6/bin:/nix/store/4i28bbf6hn8k0b1d2an6dzqmyh4vjqnm-nixpkgs-review-2.2.0/bin:/home/symphorien/bin:/run/wrappers/bin:/home/symphorien/.nix-profile/bin:/etc/profiles/per-user/symphorien/bin:/nix/var/nix/profiles/default/bin:/run/current-system/sw/bin:/home/symphorien/.fzf/bin" "/nix/store/nkmf179c1v5vis7v95dv05vi7r3sk28g-tellico/bin, /nix/store/wfmrgv16wr5ghxidiasi1cm69bghprd5-qtbase-5.12.7/bin"
```
but I assume this is because I'm not using plasma.

File: pkgs/applications/misc/tellico/default.nix [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+, libcdio
+, poppler
+, makeWrapper
+, kdoctools
+}:
+
+mkDerivation rec {
+  name = "tellico";
+  version = "3.3.0";
+  pversion = "3.3";
```
Comment (Lassulus) [inline pkgs/applications/misc/tellico/default.nix:26]: hmm, this probably not be picked up by the automatic updates. 
maybe something like `"${lib.versions.major version}.${lib.versions.minor version}"` would be better?

Comment (Lassulus) [reply pkgs/applications/misc/tellico/default.nix:26]: oh even better, there is `lib.versions.majorMinor`

**Pull request merged**